src/supertux/game_session.cpp

```diff
@@ -264,7 +264,7 @@ GameSession::draw(DrawingContext& context)
   if(game_pause)
     draw_pause(context);
 }
-
+                                                                                                                                                                                                                                                                                    
```

Comment:
What happened here?  Please remove this whitespace.

Reply:
Whoops, my bad. I'll be pushing up a new commit.